sway/desktop/transaction.c

```diff
@@ -395,6 +395,10 @@ static bool should_configure(struct sway_node *node,
 	// Xwayland views are position-aware and need to be reconfigured
 	// when their position changes.
 	if (node->sway_container->view->type == SWAY_VIEW_XWAYLAND) {
+		if (cstate->fullscreen_mode != FULLSCREEN_NONE
```

Comment:
Doesn't `content_x` change anyway in this case?

Reply:
Ya, I think I just got confused while testing something